src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/resample/resample_kernel_opt.cpp

```diff
@@ -116,17 +139,33 @@ bool ResampleKernelOpt::Validate(const Params& p, const optional_params& o) cons
         return false;
 
     const auto& input = params.inputs[0];
+    const auto & output = params.outputs[0];
 
     if ((input.GetDType() == Datatype::UINT8 || input.GetDType() == Datatype::INT8) &&
         params.resampleType != ResampleType::NEAREST_NEIGHBOR &&
-        params.resampleType != ResampleType::BILINEAR_INTERP)
+        params.resampleType != ResampleType::BILINEAR_INTERP &&
+        params.resampleType != ResampleType::LINEAR_ONNX)
         return false;
 
     if (input.GetLayout() != DataLayout::fs_b_yx_fsv32 &&
         input.GetLayout() != DataLayout::b_fs_yx_fsv16 &&
         input.GetLayout() != DataLayout::b_fs_yx_fsv32 &&
         input.GetLayout() != DataLayout::bs_fs_yx_bsv32_fsv16 &&
-        input.GetLayout() != DataLayout::bs_fs_yx_bsv32_fsv32)
+        input.GetLayout() != DataLayout::bs_fs_yx_bsv32_fsv32 &&
+
+        input.GetLayout() != DataLayout::b_fs_zyx_fsv16 &&
+        input.GetLayout() != DataLayout::bs_fs_zyx_bsv16_fsv16 &&
+        input.GetLayout() != DataLayout::bs_fs_zyx_bsv32_fsv16 &&
+        input.GetLayout() != DataLayout::b_fs_zyx_fsv32 &&
+        input.GetLayout() != DataLayout::bs_fs_zyx_bsv16_fsv32 &&
+        input.GetLayout() != DataLayout::bs_fs_zyx_bsv32_fsv32)
+        return false;
```

Comment:
i think this if is not needed at all as it duplicates
EnableInputLayout logic

Reply:
yes, you right